templates/rag-redis/pyproject.toml

```diff
@@ -33,6 +33,12 @@ langchain-cli = ">=0.0.15"
 export_module = "rag_redis.chain"
 export_attr = "chain"
 
+[tool.templates-hub]
+use-case = "rag"
+author = "Redis"
+integrations = ["OpenAI", "Redis"]
```

Comment:
@efriis this template also uses HuggingFace for embeddings. Is that a supported integration option I can add here?

Reply:
Yes!